vae/main.py

```diff
@@ -13,28 +13,31 @@
                     help='input batch size for training (default: 128)')
 parser.add_argument('--epochs', type=int, default=10, metavar='N',
                     help='number of epochs to train (default: 10)')
-parser.add_argument('--no-cuda', action='store_true', default=False,
-                    help='disables CUDA training')
-parser.add_argument('--no-mps', action='store_true', default=False,
-                        help='disables macOS GPU training')
+parser.add_argument('--accel', action='store_true', 
```

Comment:
@eromomon : I did not see this PR before it got merged. But you've changed the default in this  example. Previously acceleration was always used (if available) and there was a flag to disable it. Now default is to not use acceleration and there is a flag to enable it. I personally like new behavior better, but **we need to change the CI script** to reflect that and that was not done:
https://github.com/pytorch/examples/blob/54e132ece5558f2b54f3f57f323bcdd71525436d/run_python_examples.sh#L148